plugins/configuration.rb

```diff
@@ -145,7 +166,7 @@ def render(context)
 
       <<~MARKUP
         <div class="config-vars">
-          <h3><a class="title-link" name="configuration-variables" href="#configuration-variables"></a> Configuration Variables</h3>
+          <h4><a class="title-link" name="configuration-variables" href="#configuration-variables"></a> Configuration Variables</h4>
```

Comment:
This is always a H3, as the configuration heading is always H2. For SEO structure it is important to have that matching.

Reply:
We should decrease the size and boldness of each heading eventually